hr_employee_firstname/models/hr.py

```diff
@@ -0,0 +1,81 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2010 - 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class hr_employee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.cr_context
+    def _auto_init(self, cr, context=None):
+        res = super(hr_employee, self)._auto_init(cr, context=context)
+        cr.execute("""
```

Comment:
Please explain what this query does in docstring and explain why it cannot be done in a "create" method. Also, add a couple unit tests that check exactly what it does.